hbase-asyncfs/src/main/java/org/apache/hadoop/hbase/io/asyncfs/FanOutOneBlockAsyncDFSOutput.java

```diff
@@ -511,6 +527,7 @@ private void flush0(CompletableFuture<Long> future, boolean syncBlock) {
     return future;
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
Which future do we ignore in this method?

Reply:
Removed the annotation, let's see the code style check results. Thanks.

Reply:
Oh, no check style issues now...